packages/peregrine/src/Router/MagentoRouteHandler.js

```diff
@@ -77,6 +79,13 @@ export default class MagentoRouteHandler extends Component {
             location: { pathname }
         } = this.props;
 
+        // Depending on the environment, the fetchRootComponent global can be
+        // either an ES module with a `default` property or a plain CJS module.
+        const fetchRoot =
+            'default' in fetchRootComponent
+                ? fetchRootComponent.default
+                : fetchRootComponent;
```

Comment:
Consider checking for `__esModule` instead of `default`. (?) I'm pretty sure that's the convention.

Reply:
I just Googled a bit and I haven't found anything conclusive about that, but __esModule seems like an internal Babel thing to me.

Reply:
My mistake. I thought this was inside a plugin, but it's just inside a component.